File: Data Structures & Algorithms/max-consecutive-ones/submission-0.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        maxCnt = 0
        cnt = 0
        for i in range(len(nums)):
            if nums[i] == 1:
                cnt += 1
            else:
                cnt = 0
            maxCnt = max(cnt, maxCnt)
        return maxCnt

File: Data Structures & Algorithms/longest-substring-without-duplicates/submission-0.cpp
class Solution {
public:
    int lengthOfLongestSubstring(string s) {
        unordered_set<char> seen;
        int maxcnt = 0;
        int j =0;
        for(int i =0;i<s.length();i++){
            while(seen.find(s[i]) != seen.end()){
                seen.erase(s[j]);
                j++;
            }

            seen.insert(s[i]);

            int cnt = i - j +1;
            maxcnt = max(maxcnt, cnt);
        }

        return maxcnt;
    }
};
